charts/kube-acp-stack/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kube-acp-stack
 description: kube-acp-stack collects acp charts combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster
 type: application
-version: 2.2.1-6
```

Comment:
Not sure which version should I use here. We'd like to release 2.2.1 but here it's already defined `2.2.1-6`? Any ideas?